packages/components/managed-data-table/src/data-table-settings/droppable-panel/droppable-panel.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+import { Droppable } from 'react-beautiful-dnd';
+import Spacings from '@commercetools-uikit/spacings';
+import Text from '@commercetools-uikit/text';
+import DraggableTag from './draggable-tag';
+import TagContainerEditable from './droppable-panel.styles';
+import messages from './messages';
+
+const DroppablePanel = React.memo((props) => {
+  const { formatMessage } = useIntl();
+
+  return (
+    <Droppable droppableId={props.droppableId}>
+      {(provided) => (
+        <TagContainerEditable
+          data-testid={props.droppableId}
+          ref={provided.innerRef}
+        >
+          {props.columns.length === 0 ? (
+            <Spacings.Inset scale="s">
+              <Text.Detail tone="secondary">{props.noColumnsText}</Text.Detail>
+            </Spacings.Inset>
+          ) : (
+            <>
+              {props.columns.map((column, index) => (
+                <DraggableTag
+                  key={`${column.key}-${index}`}
+                  column={column}
+                  index={index}
+                  onRemove={
+                    props.onRemove
+                      ? () =>
+                          props.onRemove([
+                            ...props.columns.slice(0, index),
+                            ...props.columns.slice(index + 1),
+                          ])
+                      : undefined
+                  }
+                  isDisabled={props.isDisabled}
+                />
+              ))}
+              {props.isSearchable && (
+                <Spacings.Inset scale="xs">
+                  <Text.Detail tone="secondary">
+                    {formatMessage(messages.useInputSearch)}
+                  </Text.Detail>
```

Comment:
This would spare you from using `useIntl` in this component :) 
```suggestion
                  <Text.Detail tone="secondary" intlMessage={messages.useInputSearch} />
```